src/explore-education-statistics-frontend/README.md

```diff
@@ -55,6 +55,8 @@ Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
+Locally you can also run `npm run start:prod` to run the production-ready bundle.
```

Comment:
It'd be best to make this stand out more and provide some more info on how the team typically uses this command (aside from deployment). So you could do something such as: 
``` 
### `npm run start:prod`
Builds the app for production. This command can also be helpful if you are experiencing flakiness when running tests locally